commons/util/src/main/java/io/github/microcks/util/openapi/OpenAPISchemaValidator.java

```diff
@@ -146,16 +144,7 @@ public static List<String> validateJson(JsonNode schemaNode, JsonNode jsonNode)
     */
    public static List<String> validateJson(JsonNode schemaNode, JsonNode jsonNode, String namespace) {
       schemaNode = convertOpenAPISchemaToJsonSchema(schemaNode);
-
-      try {
-         return JsonSchemaValidator.validateJson(schemaNode, jsonNode, namespace);
-      } catch (ProcessingException e) {
-         log.debug("Got a ProcessingException while trying to interpret schemaNode as a real schema");
-         List<String> errors = new ArrayList<>();
-         errors.add("schemaNode does not seem to represent a valid OpenAPI schema");
-         errors.add("root cause: " + e.getMessage());
-         return errors;
-      }
+      return JsonSchemaValidator.validateJson(schemaNode, jsonNode, namespace);
```

Comment:
Is the "invalid schemaNode" error included in JsonSchemaValidator.validateJson() now?